metrics/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+
+services:
+  frontend:
+    image: "mcottontensor/sps-frontend-metrics"
+    container_name: frontend
+    network_mode: "host"
+
+  buccaneerserver:
+    image: "mcottontensor/sps-buccaneer-metrics"
```

Comment:
Can this dockerfile also be included in this PR? Or if this is just a stock buc server container, can you add a comment in the yaml pointing to the relevant part of the buc repo?

Reply:
It's stock i just pushed a version with some recent changes since i noticed early in the morning that it seemed like Will hadn't pushed the changes from the night before.

Reply:
A comment saying where the dockerfile for it lives will be sufficient then :)